tests/tests_e3/net/smtp/main_test.py

```diff
@@ -21,3 +21,46 @@ def test_sendmail():
             from_addr,
             to_addresses,
             msg_as_string)
+
+def test_sendmail_onerror():
+    from_addr = 'e3@example.net'
+    to_addresses = ['info@example.net', 'info@example.com']
+    msg_as_string = 'test mail content'
+    msg_size_excceed = "A"*1200
+    result = e3.net.smtp.sendmail(
+        from_addr,
+        to_addresses,
+        msg_size_excceed,
+        ['smtp.localhost'],
+        max_size=8/1024)
+    assert result == False
```

Comment:
when comparing against False/True always use *is*. 

So here assert `result is False`